randovania/graph/world_graph.py

```diff
@@ -87,6 +87,13 @@ class WorldGraphNode:
     - EventNode: the event
     - HintNode/PickupNode: the node resource
     - DockLockNode: the dock node resources
+
+    These resources must all provide exactly 1 quantity each.
```

Comment:
does this not cause issues with e.g. pickup nodes that grant you multiple things?

Reply:
The resources granted by a pickup in a pickup node até not added to the node itself.